packages/plugins/documentation/server/services/documentation.js

```diff
@@ -16,10 +16,26 @@ module.exports = ({ strapi }) => {
       return _.get(config, 'info.version');
     },
 
+    getDocumentationSettingsPath() {
+      return path.join(strapi.dirs.extensions, 'documentation', 'config');
+    },
+
     getFullDocumentationPath() {
       return path.join(strapi.dirs.extensions, 'documentation', 'documentation');
     },
 
+    getDocumentationSettings() {
+      try {
+        return JSON.parse(
+          fs.readFileSync(
+            path.resolve(this.getDocumentationSettingsPath(), 'settings.json')
+          )
```

Comment:
I think we should use the async `readJson` function, and then I don't think we will need `JSON.parse`